src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardController.js

```diff
@@ -95,43 +63,47 @@
         model.getTemplateMetadata().setDataTableChanged(false);
     },
 
-    /**
-    * @description Moves to next wizard step
-    */
-    next: function(component, event, helper) {
+    handleButtonClick: function(component, event, helper) {
```

Comment:
Consider breaking the conditions into smaller methods. There's a lot going on here!